view/script/trial.js

```diff
@@ -17,7 +17,26 @@
         }]
     };
     var ctx = document.getElementById('trialChart').getContext('2d');
+    var differenceValue = 0;
 
+    document.getElementById('toDate').setAttribute('min', document.getElementById('fromDate').value);
+    $('#fromDate').on('change', function changeToDateValue () {
+        var fromDateValue = document.getElementById('fromDate').value;
+        var newtoDateValue = moment(fromDateValue).add(differenceValue, 'days').format('YYYY-MM-DD');
+
+        document.getElementById('toDate').value = newtoDateValue;
+    }).on('focus', function storeDateDiff (event) {
+        var oldfromDateValue = event.target.value;
+        var oldtoDateValue = document.getElementById('toDate').value;
+
+        differenceValue = moment(oldtoDateValue).diff(moment(oldfromDateValue), 'days');
```

Comment:
![capture](https://cloud.githubusercontent.com/assets/14165530/14445490/f3ec3da6-0012-11e6-9c7f-4b62dc8a86b9.JPG)
day-format attribute was used to prevent the dates from overlapping in chrome